web/backend/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express')
+const app = express()
+
+app.get('/', function (req, res) {
```

Comment:
Codacy has a fix for the issue: [Replace `'/'` with `"/"`](https://app.codacy.com/gh/42ISTblockchain/business-basket-app/pullRequest?prid=9595197)
```suggestion
app.get("/", function (req, res) {
```